pkg/download/download.go

```diff
@@ -220,13 +221,22 @@ func createConfigsFromAPI(
 		if filter {
 			continue
 		}
+		emptyfolder = false
```

Comment:
Does that actually work? 
The problem with e.g. PUBLIC synthetic locations is at the API returns them (so there will be values), but `CreateJSONConfig` filters them out and does not any files to the folder. 

The code looks like `emptyfolder` will we false in that case?

Reply:
discussed in weekly